lib/onelogin/ruby-saml/utils.rb

```diff
@@ -1,19 +1,16 @@
 module OneLogin
   module RubySaml
     class Utils
-      def self.format_cert(cert, heads=true)
-        cert = cert.delete("\n").delete("\r").delete("\x0D")
-        if cert
-          cert = cert.gsub('-----BEGIN CERTIFICATE-----', '')
-          cert = cert.gsub('-----END CERTIFICATE-----', '')
-          cert = cert.gsub(' ', '')
+      def self.format_cert(cert)
+        # don't try to format an encoded certificate
+        return cert if cert.match(/\x0d/)
 
-          if heads
-            cert = cert.scan(/.{1,64}/).join("\n")+"\n"
-            cert = "-----BEGIN CERTIFICATE-----\n" + cert + "-----END CERTIFICATE-----\n"
-          end
-        end
-        cert
+        cert = cert.gsub(/\-{5}\s?(BEGIN|END) CERTIFICATE\s?\-{5}/, "")
```

Comment:
any chance its possible to do this certificate parsing using the `OpenSSL` module instead of `gsub` and `scan`?